server/src/main/java/io/spine/server/CommandService.java

```diff
@@ -45,8 +45,13 @@
 import static io.spine.server.bus.Acks.reject;
 
 /**
- * The {@code CommandService} allows client applications to post commands and
- * receive updates from the application backend.
+ * The {@code CommandService} provides a synchronous way to post commands
+ * to the application backend.
+ *
+ * <p>This class is an implementation of a corresponding gRPC service. Hence, public API of this
+ * class is dictated by the {@linkplain CommandServiceGrpc generated code}. Despite the fact of its
+ * "publicity", it's not meant to be used directly. Use {@link io.spine.client.Client Client}
```

Comment:
This is not true. It _is_ meant to be used directly. Otherwise, users won't be able to assembly the definitions of services for their gRPC servers at server-side. 

Please rephrase.